generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/FieldSpecFactory.java

```diff
@@ -8,11 +8,10 @@
 
 import java.math.BigDecimal;
 import java.time.OffsetDateTime;
-import java.util.Collection;
-import java.util.Optional;
-import java.util.stream.Collectors;
+import java.util.regex.Pattern;
 
 public class FieldSpecFactory {
+    public static final String RIC_REGEX = "[A-Z]{1,4}\\.[A-Z]{1,2}";
```

Comment:
It might be worth discussing with the PO what range of potential RICs we want to support - I understand there are some valid RICs that either don't have a dot or start with a dot.